parser_test.go

```diff
@@ -3548,7 +3548,6 @@ func TestParser_ParseStatement(t *testing.T) {
 		{s: `DELETE`, err: `found EOF, expected FROM, WHERE at line 1, char 8`},
 		{s: `DELETE FROM`, err: `found EOF, expected identifier at line 1, char 13`},
 		{s: `DELETE FROM myseries WHERE`, err: `found EOF, expected identifier, string, number, bool at line 1, char 28`},
-		{s: `DELETE FROM "foo".myseries`, err: `retention policy not supported at line 1, char 1`},
```

Comment:
Should this case be retained, but the expected value be changed to no error?

Reply:
Looking through this block of code it appears that its just testing for errors, I don't see any that don't test for errors here specifically.

Reply:
So let's add in some invalid retention policies.  Names with illegal characters, stuff like that.